test/functional/feature_assumevalid.py

```diff
@@ -30,6 +30,7 @@
       isn't buried by at least two weeks' work.
 """
 
+from test_framework.blocktools import COINBASE_MATURITY
 from test_framework.blocktools import (
     create_block,
```

Comment:
These two `import`s should be joined into one.

Reply:
Done, thank you.